### PR TITLE
ADEN-2264 OpenX bidder

### PR DESCRIPTION
**Pull request by fraszczakszymon**
OpenX bidder (disabled) and AmazonMatchOld clean up

### CODE REVIEW

File: extensions/wikia/AdEngine/js/lookup/openXBidder.js [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+			size,
+			slotName,
+			slotPath,
+			src = 'gpt';
+
+		for (slotName in slots) {
+			if (slots.hasOwnProperty(slotName)) {
+				size = slots[slotName];
+				slotPath = [
+					'/5441', 'wka.life', '_prowrestling', 'article', src, slotName
```
Comment (nandy-andy) [inline extensions/wikia/AdEngine/js/lookup/openXBidder.js:57]: We were talking about it yesterday on daily: let's mark it with `TODO`, so in the upcoming work we'll remove the hardcoded "wiki name".

File: extensions/wikia/AdEngine/js/lookup/openXBidder.js [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+	}
+
+	function onResponse() {
+		oxTiming.measureDiff({}, 'end').track();
+		log('OpenX bidder done', 'info', logGroup);
+		var prices = win.OX.dfp_bidder.getPriceMap(),
+			slotName;
+
+		for (slotName in prices) {
+			if (prices.hasOwnProperty(slotName) && prices[slotName].price !== 't') {
```
Comment (nandy-andy) [inline extensions/wikia/AdEngine/js/lookup/openXBidder.js]: What does `'t'` mean? Timeout? What do you think abou assigning it to a descriptive variable which we'll use in this condition?

File: extensions/wikia/AdEngine/js/AdLogicPageParams.js [hunk 3]
```diff
@@ -271,9 +271,6 @@ define('ext.wikia.adEngine.adLogicPageParams', [
 		}
 
 		extend(params, decodeLegacyDartParams(targeting.wikiCustomKeyValues));
-		if (lookups) {
```
Comment (nandy-andy) [inline extensions/wikia/AdEngine/js/AdLogicPageParams.js:274]: ~~Why are we removing it?~~ Now, I get it. It was used only for old integration with Amazon, wasn't it?

File: extensions/wikia/AnalyticsEngine/AnalyticsProviderOpenXBidder.php [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+		}
+
+		$called = true;
+
+		if ( !self::isEnabled() ) {
+			return '';
+		}
+
+		if ( $wgAdDriverEnableOpenXBidder ) {
+			$newScript = self::getIntegrationScript( 'openXBidder', 'wgAdDriverOpenXBidderCountries' );
```
Comment (nandy-andy) [inline extensions/wikia/AnalyticsEngine/AnalyticsProviderOpenXBidder.php]: `$newScript` -> `$script` or `$oxScript` ;)

File: extensions/wikia/AdEngine/js/AdLogicPageParams.js (hunk 3 above)
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/AdLogicPageParams.js:274]: Yup, it's removed here:
https://github.com/Wikia/app/pull/8492/files#diff-21e971af0f2286c20f3673208921310aL97

File: extensions/wikia/AnalyticsEngine/AnalyticsProviderOpenXBidder.php [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+		} else {
+			$newScript = '/* new integration disabled */';
```
Comment (nandy-andy) [inline extensions/wikia/AnalyticsEngine/AnalyticsProviderOpenXBidder.php]: `'/* new integration disabled */'` -> `'/* OpenX Bidder integration disabled */'`

File: extensions/wikia/AdEngine/js/lookup/services.js [hunk 6]
```diff
@@ -63,56 +61,17 @@ define('ext.wikia.adEngine.lookup.services', [
...
-			}
-		}
-
-		return params;
-	}
 
-	// No longer needed when AmazonOld is removed
-	function extendPageTargeting(pageTargeting) {
-		var amazonParams;
+		if (oxBidder && oxBidder.wasCalled() && Object.keys) {
```
Comment (nandy-andy) [inline extensions/wikia/AdEngine/js/lookup/services.js]: This code (condition and inner loop) is very similar to the code used for Amazon. I wonder if we could extract it somehow and make it less DRY.

**Comment by nandy-andy** [comment]
LGTM :+1:

**Pull request merged**